ballerina/types.bal

```diff
@@ -259,3 +259,587 @@ public enum LogLevel {
     @display {label: "OFF"}
     OFF
 }
+
+# SQL Rule
+#
+# + filter - Represents a filter which is a composition of an expression and an action that is executed in the pub/sub pipeline.
+# + action - Represents set of actions written in Sql language-based syntax
+@display {label: "SQL Rule"}
+public type SqlRule record {
+    string filter;
+    string action;
+};
```

Comment:
Missing new linw

Reply:
Addressed in [af47f42](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/180/commits/af47f42e025c870c173c07d46a984ed796dd7ab1).